### PR TITLE
Add button to admin ui

### PR DESCRIPTION
**Pull request by DblK**
This PR is only for adding a "Create New Organization" in the admin UI panel (see screenshot bellow).

Before:
image

After:
image

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1134,6 +1134,7 @@ orgs.org_manage_panel = Organization Manage Panel
 orgs.name = Name
 orgs.teams = Teams
 orgs.members = Members
+orgs.new_orga = Create New Organization
```
Comment (pgaskin) [inline options/locale/locale_en-US.ini]: It should be "Create Organization". Create implies the word new.

Comment (DblK) [reply options/locale/locale_en-US.ini]: Okay will update the ini file in next commit

File: options/locale/locale_fr-FR.ini [hunk 2]
```diff
@@ -309,7 +309,7 @@ password_change_disabled=Les utilisateurs non-locaux n'ont pas le droit de modif
 emails=Adresses e-mail
 manage_emails=Gérer les adresses e-mail
 email_desc=Votre adresse e-mail principale sera utilisée pour les notifications et d'autres opérations.
-primary=Principale
+primary=Principalef
```
Comment (sapk) [inline options/locale/locale_fr-FR.ini]: Probably a typo.
Futhermore, only  en-US should be updated the rest will be updated via crowdin.com

Comment (lunny) [reply options/locale/locale_fr-FR.ini]: ^

Comment (DblK) [reply options/locale/locale_fr-FR.ini]: Do you want me to remove all changes made to locale_fr-FR.ini?

Comment (sapk) [reply options/locale/locale_fr-FR.ini]: yes

Comment (sapk) [reply options/locale/locale_fr-FR.ini]: And don't hesitate to pass on crowdin to complete :fr: translation. ;-)

**Comment by sapk** [comment]
LGTM

**Comment by pgaskin** [comment]
LGTM

**Pull request merged**